### ISSUE TITLE
Botw Dynamic Res Missing

### ISSUE THREAD

**Issue by HamzaYslmn** [theboy181/switch-ptchtxt-mods#45]
![image](https://github.com/theboy181/switch-ptchtxt-mods/assets/78810304/0bca41bc-ffc8-4e6c-b4c3-57d02c92940a)
![image](https://github.com/theboy181/switch-ptchtxt-mods/assets/78810304/8ec7f5b3-c769-4386-8661-ef2fee089886)

**Comment by Nashismo** [comment]
Exactly!! I have been trying to find this one for so long! And once I finally found it, the creator has removed it! So yes, I confirm it: IT CANNOT BE DOWNLOADED, the link is dead.

Wait! I just found it in the archive 2021, here: https://github.com/yuzuMods/yuzuMods/blob/2d3b70402ee92eb333556d75046e6aae819a1d09/Disable-Dynamic-Resolution.rar

**Comment by HamzaYslmn** [comment]
[Disable Dynamic Resolution.zip](https://github.com/theboy181/switch-ptchtxt-mods/files/14469067/Disable.Dynamic.Resolution.zip)
I found it too :D

**Comment by theboy181** [comment]
Why do we still need this mod?
